neqo-http3-server/src/main.rs

```diff
@@ -375,8 +378,17 @@ fn main() -> Result<(), io::Error> {
 
     let mut events = Events::with_capacity(1024);
 
+    let mut active_servers: HashSet<usize> = HashSet::new();
+
     loop {
-        poll.poll(&mut events, None)?;
+        poll.poll(
+            &mut events,
+            if active_servers.is_empty() {
+                None
+            } else {
+                Some(Duration::from_millis(0))
+            },
+        )?;
```

Comment:
I assume that you want to poll here to pick up any datagrams that arrived while we were processing stuff from the last round.  It might be worth documenting that.